packages/core/__tests__/channel-recipes.js

```diff
@@ -37,7 +37,11 @@ test('action channel', () => {
 })
 
 test('error check when constructing actionChannels', () => {
-  const middleware = sagaMiddleware()
+  const middleware = sagaMiddleware({
+    onError: err => {
+      expect(err.message).toMatchInlineSnapshot(`"actionChannel(pattern,...): argument pattern is not valid"`)
```

Comment:
probably we should remove error message assertion below

Reply:
That's true - it could be removed, but I didnt want to change too much here. The proper way would be to remove latter assertion AND making sure that this one is happening (probably by wrapping the whole thing in `return new Promise(...)` and resolving it in `onError`.

This could be done, but as I've said - I didn't want to change too much, I just finally got annoyed by messy output which has been produced and I've fixed it in the fastest way I could.